### ISSUE TITLE
[REQUEST]  Auto-loot Toggle right click (use) feature with clams, etc.

### ISSUE THREAD

**Issue by rbpaiett** [niketa-wow/farming-bar#15]
When right-click is used for items that open (clams, etc.) The window hangs open.  If I were to use shift with it, instead of auto-looting, the item will disappear due to the remove feature from the addon.  ( I like using the use feature for items that can't be put on a regular toolbar).

Perhaps a toggle feature in the options could make it so it auto-loots upon use (problem is not all items open up like clams or chests do so I don't know how difficult the programming would be).

The alternative is to leave it as it since this is not a big deal.  It's a nice-to-have, not a must-have.

I look forward to your thoughts on the matter.

**Comment by niketa-wow** [comment]
I'll have to look into it a bit and see what I can do. I haven't encountered this yet, so I'm not sure exactly all that would be involved. I'd have assumed if you have auto loot on in general it should already, but my concern is whether or not looting would be protected or not if I do have to forcefully loot.

**Comment by niketa-wow** [comment]
Do you have auto loot turned off normally and you're wanting to auto loot when using from the bar?

**Comment by rbpaiett** [comment]
I normally have auto loot disabled so I use shift when I need it. With clams if you shift click them in your inventory, it opens and you auto loot it. That's the mechanics in looking for. I don't have auto loot enabled because I want to pick and choose when needed.

**Comment by niketa-wow** [comment]
What specific clam were you trying to open? I didn't have that problem when I checked quickly in retail or classic, even after turning off auto loot.

Though I may need to check again with other add-ons disabled, but wanna make sure I'm testing the right clam. 

On classic I went out to get a barnacled clam and it was working fine. But I'm only level 39 since I primarily play retail so if it is a higher level idk if I can test it myself.
